### ISSUE TITLE
Image not displaying

### ISSUE THREAD

**Issue by depperm** [miaolz123/vue-markdown#94]
I am trying to display an image and it isn't displaying. Other vue-markdown content displays fine. The location is correct as right above it I've added an img

    <img src="@/assets/7801_4_lang_image.png">
    <vue-markdown>![img](@/assets/7801_4_lang_image.PNG)</vue-markdown>

Changing the `@` to `..` doesn't change anything either, the top one displays but the bottom one doesn't. Am I referencing the image wrong? Why won't it display?

**Comment by ideacco** [comment]
Yes, I also found that I could not display the image, so what should be the correct image path in DEV mode and build mode?
